fj-host/src/platform.rs

```diff
@@ -0,0 +1,45 @@
+// Represents platform trait
+trait Platform {
+    fn file_name(name: &str) -> String;
```

Comment:
It might be a good idea to give this a more descriptive name. Maybe `model_lib_file_name`?